src/views/Home/Home.tsx

```diff
@@ -35,8 +35,7 @@ const Home: React.FC = () => {
 			</div>
 			<Spacer size="lg" />
 			<StyledInfo>
-				🏆<b>Pro Tip</b>: BAO-ETH UNI-V2 LP token pool yields 32x more tokens
-				than other pairs.
+				🏆<b>Pro Tip</b>: BAO-ETH UNIV2 LP token pool is weighted at 55000, the biggest of all pairs.
```

Comment:
Could we change this to:

"55x the base rewards (already factored into displayed APY)"

The weighting at 55000 might be a bit confusing to some.

Reply:
Sure, I just think 55x is misleading without a proper explanation.

Reply:
Maybe we can also link to the pool weights section of the docs as well?

Reply:
Good, I will open another PR with that!